docs/content/rules/sort-union-types.mdx

```diff
@@ -201,7 +201,20 @@ type Example =
                   type: 'alphabetical',
                   order: 'asc',
                   ignoreCase: true,
-                  groups: [],
+                  groups: [
+                    'conditional',
```

Comment:
We specify default values everywhere in the examples. Maybe it is unnecessary?

Reply:
I reverted the `Usage` section to what it was and updated the example section directly instead.